app/helpers/cart_helper.rb

```diff
@@ -0,0 +1,33 @@
+module CartHelper
+  def get_name_product pro_id
+    product = Product&.find_by(id: pro_id)
+    product.name
+  end
+
+  def get_price_product pro_id
+    product = Product&.find_by(id: pro_id)
+    product.price
+  end
+
+  def get_category_product pro_id
+    product = Product&.find_by id: pro_id
+    product.category_name
+  end
+
+  def count_item
+    @count = 0
+    @carts.each do |(_pro_id, qty)|
```

Comment:
nghiên cứu thêm sử dụng hàm .sum hơn là for each như này

Reply:
thần kỳ quá anh ơi